Makefile

```diff
@@ -18,9 +19,13 @@ fmt:
 test:
 	@go test -v ./...
 
-
 # http://godoc.org/code.google.com/p/go.tools/cmd/vet
 # go get code.google.com/p/go.tools/cmd/vet
 vet:
 	go vet ./...
 
+$(CODEGEN_JAR):
+	wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O $(CODEGEN_JAR)
+
+server-api: ./v1alpha1/grafeas.json $(CODEGEN_JAR)
+	java -jar $(CODEGEN_JAR) generate -l go -i ./v1alpha1/grafeas.json -o ./server-go/api  -D packageName=api
```

Comment:
We probably want to version this - so that we can support new versions of the model in the future.

Reply:
right. I'll make this `./server-go/api/v1alpha1`